pkg/kubectl/cmd/get.go

```diff
@@ -382,5 +395,9 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			continue
 		}
 	}
+	w.Flush()
+	if printer != nil {
+		printer.FinishPrint(os.Stderr, lastMapping.Resource)
```

Comment:
You should not use `os.Stderr` directly, but instead get an `errout` io.Writer passed in constructor by the code that initializes the command.